packages/expo-permissions/src/PermissionsHooks.ts

```diff
@@ -23,6 +23,7 @@ export function usePermissions(
   type: PermissionType | PermissionType[],
   options: PermissionsOptions = {}
 ): [PermissionResponse | undefined, () => Promise<void>, () => Promise<void>] {
+  const isMounted = useRef(true);
```

Comment:
```suggestion
  const isMounted = useRef(true);
```
Why not to go with plain `useState<boolean>(false)` here? 🤔

Reply:
cuz that will cause the state to update, a ref doesn't.